event_hello/src/test.rs

```diff
@@ -0,0 +1,12 @@
+#![cfg(test)]
+
+use super::*;
+use soroban_sdk::{BytesN, Env};
+
+#[test]
+fn test() {
+    let env = Env::default();
+    let contract_id = BytesN::from_array(&env, &[0; 32]);
+    env.register_contract(&contract_id, EventContract);
+    EventContractClient::new(&env, &contract_id).hello(&Symbol::from_str("SourBun"));
```

Comment:
Is there a way to assert on the events at all? If not no worries, although we should create an issue for that.

Reply:
There isn't. It either successes returning empty or traps on failure. What kind of assertion do you have in mind?

Reply:
We just need to be able to assert on an event having been published. This isn't in the SDK yet though, so nothing to do here. I opened an issue about it: https://github.com/stellar/rs-soroban-sdk/issues/532.